daffodil-runtime2/src/main/scala/org/apache/daffodil/runtime2/generators/CodeGeneratorState.scala

```diff
@@ -247,10 +264,12 @@ class CodeGeneratorState {
     val erds = this.erds.mkString("\n")
     val finalImplementation = this.finalImplementation.mkString("\n")
     val code =
-      s"""#include "generated_code.h" // for generated code structs
-         |#include <endian.h>         // for be32toh, htobe32, etc.
-         |#include <stddef.h>         // for ptrdiff_t
-         |#include <stdio.h>          // for NULL, fread, fwrite, size_t, FILE
+      s"""#define __STDC_WANT_IEC_60559_BFP_EXT__
```

Comment:
Comment needed to explain what WANT_IEC_60559_BFP means. I think BFP stands for binary floating point?

Reply:
I needed this WANT_IEC_60559_BFP to make <math.h> provide the signaling NaN macros (SNAN, SNANF), but I ended up removing that define in my second commit because it didn't work on Windows.  I decided it wasn't worth trying to initialize fields with SNAN/SNANF (too unportable) and initialized them with NAN instead (a more portable macro).